dossierfacile-common-library/src/main/java/fr/dossierfacile/common/entity/Tenant.java

```diff
@@ -92,12 +93,14 @@ public class Tenant extends User implements Serializable {
     public Tenant(String email) {
         super(email);
         this.tenantType = TenantType.CREATE;
+        this.userType = TENANT_TYPE;
```

Comment:
Il faudrait peut-etre supprimer l'annotation 'at'AllArgsconstructor (y compris sur User)?
"userType" devrait peut-etre être mis dans le constucteur User car ce champ est sur user.

Nice to Have: Sauf erreur de ma part, le constructeur User n'est utilisé que pour le BO. => User pourrait être abstrait et on pourrait avoir une class BOUser. Après il y a peut-etre qq chose a creuser au niveau des ROLES avant d'engager ça.

Reply:
j'ai corrigé en suivant tes premières remarques, je garde en tête le fait de passer user_account en abstract mais juste je le sens pas de le faire là